test/db/cmd/cmd_open

```diff
@@ -105,8 +105,7 @@ bins/elf/crackme0x00b
 EOF
 CMDS=oij
 EXPECT=<<EOF
-{"raised":true,"fd":3,"uri":"bins/elf/crackme0x05","from":0,"writable":true,"size":7656}
-{"raised":true,"fd":5,"uri":"bins/elf/crackme0x00b","from":0,"writable":true,"size":7375}
+[{"raised":false,"fd":3,"uri":"bins/elf/crackme0x05","from":0,"writable":false,"size":7656},{"raised":true,"fd":5,"uri":"bins/elf/crackme0x00b","from":0,"writable":false,"size":7375}]
```

Comment:
don't forget to also add a json test in https://github.com/radareorg/radare2/tree/master/test/db/json

Reply:
Hey, I have added that:

https://github.com/radareorg/radare2/pull/17642/files#diff-11f08680e054bf1235c7c156ec3962f4272ae558b64c44158f965f1d287cbf49

Reply:
Marking as resolved.